CommonAssemblyInfo.cs

```diff
@@ -1,7 +1,9 @@
 ﻿using System.Reflection;
 using System.Runtime.InteropServices;
 
+#if NET6_0_OR_GREATER
```

Comment:
I believe you are not going to back-port to earlier versions, i.e. superfluous. Or do you intend to remove Win7?

Reply:
The source generators are targeting .NET Standard 2.0, which is incompatible with this. Hence the guard.